solidity/dashboard/src/components/coverage-pools/ExchangeRate.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import { Keep } from "../../contracts"
+import TokenAmount from "../TokenAmount"
+
+const BaseExchangeRate = ({
+  covToken,
+  amount,
+  htmlTag = "div",
+  className = "",
+}) => {
+  const Tag = htmlTag
+  return (
+    <Tag className={className}>
+      {`1 ${covToken.symbol}`} = ~
+      <TokenAmount
+        amount={amount}
+        decimalsToDisplay={3}
+        wrapperClassName="flex-inline"
+        amountClassName=""
+        symbolClassName=""
+      />
+    </Tag>
+  )
+}
+
+export const CoveragePoolV1ExchangeRate = ({
+  covToken,
+  collateralToken,
+  covTotalSupply,
+  totalValueLocked,
+  ...restProps
+}) => (
+  <BaseExchangeRate
+    {...restProps}
+    covToken={covToken}
+    amount={Keep.coveragePoolV1.estimatedBalanceFor(
```

Comment:
Can we round the value returned from `estimatedBalanceFor`? 

The problem with the function is that it can return value that has more than 18 decimal places. For example:

1.000000000000000000000000000345

Because of that `TokenAmount` will print such numer as 1.000 (1.000000000000000000 in the tooltip)

We should cut out any decimals that are less than the smallest KEEP unit when calculating the estimatedBalance. So i would add `decimalPlaces` to the function:

```
return new BigNumber(covKeep)
      .div(totalSupplyOfCovKeep)
      .multipliedBy(tvl)
      .decimalPlaces(KEEP.smallestPrecisionDecimals)
      .toString()
```